absl/strings/escaping_test.cc

```diff
@@ -635,4 +635,99 @@ TEST(HexAndBack, HexStringToBytes_and_BytesToHexString) {
   EXPECT_EQ(hex_only_lower, hex_result);
 }
 
+TEST(NonNullTerminatedStrings, OctalConversion) {
```

Comment:
These tests all test absl::CUnescape, so let's make them look like the other absl::CUnescape tests.  Let's put them up with the other CUnescape tests and have them start with `TEST(CUnescape`, like this:

```
TEST(CUnescape, NonNullTerminatedOctalConversion) {
  ...
```

Reply:
Makes sense, will do :)